src/endpoints/Newsrooms/types.ts

```diff
@@ -30,7 +30,7 @@ export interface CreateRequest {
      * {subdomain}.prezly.com
      */
     subdomain: string;
-    cultures: CultureRef['code'][];
+    cultures?: CultureRef['code'][];
```

Comment:
I'm not sure why it was required before (probably an oversight) as it's possible to create a newsroom without specifying them.